ff-macros/src/montgomery/biginteger.rs

```diff
@@ -57,8 +57,8 @@ pub(super) fn subtract_modulus_impl(
 ) -> proc_macro2::TokenStream {
     quote! {
         #[inline(always)]
-        fn __subtract_modulus(a: &mut F) {
```

Comment:
To avoid possible slowdowns for existing moduli, maybe we should introduce two variants of the relevant methods?

Reply:
We know which method to call statically, since we know where |MODULUS| = 256 bits

Reply:
If you feel that there won't be slowdowns, it would be good to validate this with some benchmarks.

Reply:
Some changes have been made, so that for automatic derivation it would choose the right one.